VRCFaceTracking/UnifiedConfig.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using VRCFaceTracking.Assets.UI;
+using VRCFaceTracking.Params;
+
+namespace VRCFaceTracking.Types
+{
+    public class UnifiedConfig
+    {
+        private static string unifiedConfigPath = Utils.PersistentDataDirectory + "\\Config.json";
+        private static JsonSerializerOptions jsonSerializerOptions = new JsonSerializerOptions() { WriteIndented = true, IncludeFields = true, DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull };
+
+        public List<string> RequestedModulePaths = new List<string>();
+        public UnifiedTrackingData Data = new UnifiedTrackingData();
+        public UnifiedTrackingMutator Mutator;
+
+        public void ReadConfiguration()
+        {
+            UnifiedConfig config = new UnifiedConfig();
+            try
+            {
+                string jsonString = File.ReadAllText(unifiedConfigPath);
+                config = JsonSerializer.Deserialize<UnifiedConfig>(jsonString, jsonSerializerOptions);
+                if (config != null)
+                {
+                    if (config.RequestedModulePaths != null && config.RequestedModulePaths.Count > 0)
+                    {
+                        Logger.Msg("Saved module load order initialized.");
+                        UnifiedLibManager.RequestedModules = UnifiedLibManager.LoadExternalAssemblies(config.RequestedModulePaths.ToArray());
+                    }
+                    else Logger.Msg("Load order not found; initializing default order scheme.");
+
+                    if (config.Mutator != null)
+                    {
+                        Logger.Msg("Saved data mutation settings loaded.");
+                        UnifiedTracking.Mutator = config.Mutator;
+                    }
+                    else Logger.Msg("Mutators not found; initializing default mutator.");
+                }
+            }
+            catch (FileNotFoundException)
+            {
+                Logger.Warning("Configuration file not found; creating placeholder 'Config.json' file.");
+                Save();
+            }
+            catch (JsonException)
+            {
+                Logger.Warning("Configuration file has been corrupted, purging file.");
+                File.Delete(unifiedConfigPath);
+            }
```

Comment:
Specifically for the Config file, I added this early on, however completely purging the file is obviously not ideal if we expect users to be able to tune their data continuously. I am not entirely familiar with Text.Json, but I imagine there is a way to allow it to still propogate data even if the objects are not identical (potentially in the case of future updates to VRCFT) and retain configs thru updates.